sources/packages/backend/libs/services/src/msfaa-number/msfaa-number-shared.service.ts

```diff
@@ -57,6 +61,34 @@ export class MSFAANumberSharedService {
     );
   }
 
+  /**
+   * Creates a new MSFAA number from the SFAS MSFAA number
+   * for the particular offering intensity and also activating the
+   * new MSFAA number for the same offering intensity.
+   * @param sfasMSFAANumber SFAS MSFAA number.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * individually based on, for instance, the Part time/Full time.
+   * @returns created and activated MSFAA record.
+   */
+  async activateMSFAANumber(
```

Comment:
Do we still need this method since we now have `importMSFAAumberFromSFAS`?
I would recommend moving the code from here to `importMSFAAumberFromSFAS` since this one is just passing the same parameter over.